src/freenet/support/WrapperKeepalive.java

```diff
@@ -14,7 +15,7 @@
   public void run() {
     while (!shutdown) {
       try {
-        WrapperManager.signalStarting(INTERVAL);
+        WrapperManager.signalStarting(INTERVAL + (int)SECONDS.toMillis(5));
```

Comment:
The magic value comes out of thin air... and sounded reasonable at the time; arguably it could be smaller.

I don't know of any better way to schedule it regularly... we could pass a reference to the node's Ticker... but it's pretty deep in the stack... and I've figured that simplicity was better here (after all the thread dies when we leave the try{} block!)